composer/trainer/trainer_hparams.py

```diff
@@ -469,9 +475,6 @@ def validate(self):
 
     def initialize_object(self) -> Trainer:
         self.validate()
-        import composer
-        logging.getLogger(composer.__name__).setLevel(self.log_level)
```

Comment:
Removed this parameter -- was thinking it would be too confusing with the console log level parameter below

Reply:
(Adding this parameter back in, but going to call it python_log_level)